test/python/lcm_udpm_queue_issue_test.py

```diff
@@ -0,0 +1,81 @@
+import hashlib
+import os
+import unittest
+import time
+
+from lcm import LCM
+
+
+def lcm_handle_all(lcm, timeout=0.01):
+    total_count = 0
+    while True:
+        count = lcm.handle_timeout(timeout)
```

Comment:
This doesn't work with the latest version of LCM / Python3. `handle_timeout` wants an integer representing the number of milliseconds rather than a float. If this used to be seconds, than an equivalent would be

```suggestion
def lcm_handle_all(lcm, timeout=10):
    total_count = 0
    while True:
        count = lcm.handle_timeout(timeout)
```